# Human Pose Topic 
human_pose_topic: '/HumanPose'
# Human Path Topic 
human_path_topic: '/HumanPath'

# UAV Pose Topic 
# uav_pose_topic: '/UAV1Pose'
# UAV Command Velocity Topic
cmd_vel_topic: 'follow_me_velocity_smoother/raw_cmd_vel'
# UAV Following Distance
#follow_distance: 1.0  # m

# Activation Service
activation_service: 'follow_me/enable'

#####################################################

# Feedback Controller Gains
angular_Kp: 1.00
angular_Kd: 0.00
angular_Ki: 0.06
max_angular_velocity: 2     # rad/s

linear_velocity_gain: 2     # m/s
linear_velocity_attenuation: 2 
max_linear_velocity: 0.7    # m/s
distance_tolerance: 0.4    # m


# Obstacle Avoidance Parameters
obstacle_range: 1.0
avoidance_gain: 0.5
obs_vel_attenuation: 10

# UAV Sonar Topics
sonar_left: 'sonar/sonar_left'
sonar_mid: 'sonar/sonar_mid'
sonar_right: 'sonar/sonar_right'
